src/interpreter/ByteCode.h

```diff
@@ -84,6 +84,15 @@ class FunctionType;
     F(Load64)                   \
     F(Store32)                  \
     F(Store64)                  \
+    F(MemorySizeMulti)          \
```

Comment:
I think these are rare operations, we could just add multi memory support for them.